cirq/linalg/transformations.py

```diff
@@ -149,6 +149,45 @@ def targeted_left_multiply(left_matrix: np.ndarray,
                      **({'out': out} if out is not None else {}))
 
 
+def targeted_conjugate(
+    tensor: np.ndarray,
+    target: np.ndarray,
+    indices: Sequence[int],
+    conj_indices: Sequence[int],
```

Comment:
Why aren't these redundant with the non-conjugated indices indices?

Reply:
because it is more general (you don't have to match with len(target.shape)/2 + indices

Reply:
Can they be inferred by default to do the simple case?